### PR TITLE
Fail the build if kubelet isn't labeled correctly

### PR DESCRIPTION
**Pull request by cgwalters**
See https://github.com/coreos/coreos-assembler/issues/1958
This lets us find the problem cheaply here instead of much
later in cluster CI.

### CODE REVIEW

File: entrypoint.sh [hunk 1]
```diff
@@ -150,3 +150,8 @@ popd
 
 # build new commit
 coreos-assembler dev-overlay --repo /srv/repo --rev "${REF}" --add-tree /tmp/working --output-ref "${REF}"
+ostree --repo=/srv/repo ls -X "${REF}" /usr/bin/kubelet > /tmp/working/label.txt
+if ! grep -q ':bin_t:' /tmp/working/label.txt; then
+  echo "error: Invalid label on kubelet"
+  cat /tmp/working/label.txt
```
Comment (jlebon) [inline entrypoint.sh:156]: Shouldn't we e.g. `exit 1` here?

Comment (cgwalters) [reply entrypoint.sh:156]: Heh yes, thanks fixed!

**Comment by jlebon** [comment]
Makes sense to me!

/approve

**Pull request merged**